moped-editor/src/components/GridTable/Filters.js

```diff
@@ -278,6 +282,9 @@ const Filters = ({
    * Applies the current local state and updates the parent's state
    */
   const handleApplyButtonClick = () => {
+    if (searchFieldValue) {
+      setSearchFieldValue(searchFieldValue.trim());
+    }
```

Comment:
@frankhereford  Thanks for updating this in here! I think this is actually just a few lines away from fixing https://github.com/cityofaustin/atd-data-tech/issues/22693 too. The `search` params just needs to be set here like in the `Search` component. Then the "Apply" button will update the GraphQL query with the latest search box and advanced filter values.

What do you think about adding that fix in here?

Reply:
@mddilley wrote:

> What do you think about adding that fix in here?

I think that's an awesome idea. Who wouldn't take two birds with one stone if they get a chance. This'll be my monday thing while i'm drinking coffee.

Reply:
https://github.com/cityofaustin/atd-moped/pull/1612